.circleci/docker/ubuntu-cuda/Dockerfile

```
@@ -99,5 +98,11 @@ ENV CUDA_PATH /usr/local/cuda
 # Install LLVM dev version (Defined in the pytorch/builder github repository)
 COPY --from=pytorch/llvm:9.0.1 /opt/llvm /opt/llvm
 
+# Install CUDNN 8
+ARG CUDNN_VERSION
+ADD ./common/install_cudnn8.sh install_cudnn8.sh
+RUN if [ "${CUDNN_VERSION}" -eq 8 ]; then bash install_cudnn8.sh; fi
+RUN rm install_cudnn8.sh
+
```

Please pass CUDA_VERSION as an argument here
```suggestion
ARG CUDNN_VERSION
ARG CUDA_VERSION
ADD ./common/install_cudnn8.sh install_cudnn8.sh
RUN if [ "${CUDNN_VERSION}" -eq 8 ]; then bash install_cudnn8.sh ${CUDA_VERSION}; fi
RUN rm install_cudnn8.sh

```

---

This will be installed on cuda 11.3, 11.5 and 11.6 . Adding a check for CUDNN VERSION